src/commands/subdomain.rs

```diff
@@ -105,15 +117,27 @@ pub fn set_subdomain(name: &str, user: &GlobalUser, target: &Target) -> Result<(
     }
     let subdomain = Subdomain::get(&target.account_id, user)?;
     if let Some(subdomain) = subdomain {
-        let msg = if subdomain == name {
-            format!("You have previously registered {}.workers.dev", subdomain)
+        if subdomain == name {
+            let msg = format!("You have previously registered {}.workers.dev", subdomain);
+            message::success(&msg);
+            return Ok(());
         } else {
-            format!("This account already has a registered subdomain. You can only register one subdomain per account. Your subdomain is {}.workers.dev", subdomain)
-        };
-        failure::bail!(msg)
-    } else {
-        register_subdomain(&name, &user, &target)
+            // probably need to list all the effected scripts?
```

Comment:
yes - should have parity w/what we do in the UI. UI lists all renames with `"worker.avery.workers.dev" => "worker.natalie.workers.dev".

Reply:
Done.